Rdmp.Dicom.UI/SemEHRUI.Designer.cs

```diff
@@ -324,6 +328,10 @@ private void InitializeComponent()
             // tabPage2
             // 
             this.tabPage2.AutoScroll = true;
+            this.tabPage2.Controls.Add(this.tbApiHttpAuthPassword);
```

Comment:
Hi would you please be able to change this to store the ID of a `DataAccessCredentials` instead?  This will give us several advantages:

- Password will be encrypted in RDMP
- Password won't be replicated in every config

You can have a button and use IBasicActivateItems.SelectOne to pick it in the callback